src/ui/components/filters/facetscomponent.js

```diff
@@ -275,5 +275,6 @@ export default class FacetsComponent extends Component {
     );
 
     this._filterbox.mount();
+    this.core.globalStorage.set(StorageKeys.FACETS_LOADED, true);
```

Comment:
Is this the only place we need to set this? If I execute a new query and it returns new Facets, do we go through this code pathway again?

Reply:
Yup. every time we execute a new query, DYNAMIC_FILTERS is updated, which calls the Facets component setState, and then its onMount